staging/src/k8s.io/apiserver/plugin/pkg/audit/log/backend.go

```diff
@@ -34,53 +35,59 @@ const (
 	FormatJson = "json"
 )
 
+// The plugin name reported in error metrics.
+const pluginName = "log"
+
 // AllowedFormats are the formats known by log backend.
 var AllowedFormats = []string{
 	FormatLegacy,
 	FormatJson,
 }
 
+func NewBackend(out io.Writer, format string, groupVersion schema.GroupVersion) (audit.Backend, error) {
```

Comment:
Still don't understand why you need to return an error

Reply:
No need now. Originally , I just want to keep conaisten with webhook backend.